packages/cdc-react/src/components/Pill/Pill.scss

```diff
@@ -1,7 +1,7 @@
 @use "../../scss/global.scss";
 @use "uswds-core" as *;
 
-.cdc-react {
+.cdc-react.pill {
   display: inline-flex;
   align-items: center;
   align-items: center;
```

Comment:
Not part of your changes, but just noticed this dupe style rule.  Can you please remove?